DevModeHandlerManagerImpl not serializable

We're getting test failures due to:
`Caused by: java.io.NotSerializableException: com.vaadin.base.devserver.DevModeHandlerManagerImpl`

Example:
https://teamcity.vaadin.com/viewLog.html?buildId=83701&buildTypeId=VaadinFlowComponents_PullRequestValidation

Either make the class serializable or include it to excludedPatterns in [ClassesSerializableTest](https://github.com/vaadin/flow/blob/master/flow-test-generic/src/main/java/com/vaadin/flow/testutil/ClassesSerializableTest.java)

--- a/flow-test-generic/src/main/java/com/vaadin/flow/testutil/ClassesSerializableTest.java
+++ b/flow-test-generic/src/main/java/com/vaadin/flow/testutil/ClassesSerializableTest.java
@@ -71,6 +71,7 @@ public abstract class ClassesSerializableTest extends ClassFinder {
 
                 "com\\.vaadin\\.flow\\.internal\\.VaadinContextInitializer",
                 "com\\.vaadin\\.flow\\.internal\\.ApplicationClassLoaderAccess",
+                "com\\.vaadin\\.base\\.devserver\\.DevModeHandlerManagerImpl",
                 "com\\.vaadin\\.flow\\.data\\.provider\\.InMemoryDataProviderHelpers",
                 "com\\.vaadin\\.flow\\.di\\.InstantiatorFactory",
                 "com\\.vaadin\\.flow\\.di\\.Lookup(\\$.*)?",

